source/index.html.md

```diff
@@ -71,6 +71,21 @@ also provision other users accounts as they signup for this Venmo
 clone which will be called `AnchorX`. Although each user will have a Stellar account they won't know
 about it.
 
+[TODO: Add intro of why are you using JS and what is required dependency-wise]
+[IDEA: "Precompiler Section"]
+
+Stellar is built through a RESTful HTTP API server called Horizon. You can run your
```

Comment:
I added a section about stellar core and stellar horizon which links to the getting started pages. It also mention that we'll be using instances run by the stellar development foundation